### PR TITLE
Update 000_wc_other_titles.txt

### PR DESCRIPTION
**Pull request by IronyOwl**
## Changelog:
- Witherbark, Vilebranch, Mossflayer, and Revantusk tribes now start as vassals under Zul'Aman, but become independent upon Zul'jin's disappearance

### CODE REVIEW

File: history/titles/000_wc_other_titles.txt [hunk 1]
```diff
@@ -372,6 +372,7 @@ d_witherbark = {
 	503.7.20 = { holder = amani20 } #Ugononbu[8054]
 	539.1.22 = { holder = amani21 } #Miguza[8054]
 	573.1.1={holder=54000} #Zalas [8150]
+	573.1.1 = {liege = k_zulaman}  #united under Zul'jin
```
Comment (zumbak04) [inline history/titles/000_wc_other_titles.txt:375]: They should become free after Zul'jin disappears.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: history/titles/000_wc_other_titles.txt (hunk 1 above)
Comment (IronyOwl) [reply history/titles/000_wc_other_titles.txt:375]: It is now so.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by StarsPotato — APPROVED** [review]
Works perfectly fine!

**Pull request merged**